01_basic_stream/README.md

```diff
@@ -0,0 +1,135 @@
+# Simple stream
+
+This tutorial will get you started with the basic of using Anbox Cloud. By the end you will
+have a custom web page capable of streaming Android 10.
+
+## 1. Creating the application
+
+Create a directory and place a file named `manifest.yaml` in it.
+
+```shell
+$ mkdir android10
+$ touch android10/manifest.yaml
+```
+
+And copy the following in your manifest:
+
+```yaml
+# manifest.yaml
+name: my-android10-app
+instance-type: a2.3
+```
+
+Then use `amc` to create the application:
+
+```shell
+$ amc application create ./android10
+```
+
+`amc` will take a few minutes to prepare the application (apply security patches, optimize image size, run hooks, etc).
+You can monitor progress by running `amc application list`
+
+
+## 2. Building a web app to stream your Android applications
+
+Create a simple webpage containing an element with the ID `anbox-stream`. This is where the video stream will be displayed:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+</head>
+<body>
+    <div id="anbox-stream"></div>
+</body>
+```
+
+Then include the stream SDK:
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+    <script src="./path/to/anbox-stream-sdk.js"></script>
```

Comment:
This wont work for just copy&paste, right?

Reply:
Nope, but if we want this to work with a copy and paste, we need to include the SDK in the repo

Reply:
Why that? We can still put instructions ala `git clone ...`